solidity/contracts/middleware/InterchainAccountRouter.sol

```diff
@@ -60,17 +61,20 @@ contract InterchainAccountRouter is Router, IInterchainAccountRouter {
         view
         returns (address)
     {
-        return _getInterchainAccount(_salt(_origin, _sender));
+        bytes memory bytecode = MinimalProxy.bytecode(implementation);
```

Comment:
Why not move this line to _getInterchainAccount?

Reply:
because it's used in the `Create2.deploy`